libkbfs/root_metadata.go

```diff
@@ -316,17 +316,19 @@ func (md *RootMetadata) ReadOnly() ReadOnlyRootMetadata {
 // can be assumed to never alias a (modifiable) *RootMetadata.
 type ImmutableRootMetadata struct {
 	ReadOnlyRootMetadata
-	mdID MdID
+	mdID           MdID
```

Comment:
can you add a comment about what `localTimestamp` means? I.e. that it either comes from the the source of the RootMetadata, and represents the (untrusted) time the metadata was created, and that it's translated to the local clock.